docs/source/technical_design/containers.md

```diff
@@ -102,8 +102,7 @@ We have two container images that are used to run multiple containers. What the
 | boefjes         | normalizer  | Normalizers runtime                                                               |
 | boefjes         | katalogus   | Katalogus API                                                                     |
 | octopoes        | web         | Octopoes API                                                                      |
-| octopoes        | worker-beat | Celery worker running beat. There must only be exactly one container of this type |
```

Comment:
I think this one still exists in the current Compose project setup

Reply:
The container is called `octopoes_api_worker` and it processes events from RabbitMQ. We no longer use Celery.

Reply:
Yes the service has been renamed but there are still references to celery (as a dependency and code)